FirebaseSessions/Tests/TestApp/Shared/MockSubscriberSDK.swift

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import FirebaseSessions
+
+// Avoids exposing internal FirebaseCore APIs to Swift users.
+@_implementationOnly import FirebaseCoreExtension
+
+@objc(FIRMockSubscriberSDKProtocol)
+protocol MockSubscriberSDKProtocol {
+  func emptyProtocol()
+}
+
+@objc(FIRMockSubscriberSDK) final class MockSubscriberSDK: NSObject, Library, SessionsSubscriber,
+  MockSubscriberSDKProtocol {
+  static func addDependency() {
+    FirebaseApp.registerInternalLibrary(
+      MockSubscriberSDK.self,
+      withName: "mock-firebase-sessions-subscriber-sdk"
+    )
+    SessionsDependencies.addDependency(name: SessionsSubscriberName.Performance)
```

Comment:
So this mock sdk is pretending to be Fireperf right?

Reply:
That's right